climate-data-ca/template/hero/resource-hero-content.php

```diff
@@ -1,90 +1,132 @@
-      
-      <?php
+    <nav aria-label="breadcrumb">
+        <ol class="breadcrumb">
+            <li class="breadcrumb-item"><a href="<?php echo get_permalink ( $training_page ); ?>"><?php echo get_the_title ( $training_page ); ?></a></li>
+            
+            <?php
+      $post_cat = get_the_terms ( get_the_ID(), 'resource-category' );
+
+      if ( is_array( $post_cat ) ) {
+	      $post_cat = $post_cat[0]; ?>
+          <li class="breadcrumb-item"><a href="<?php echo get_permalink( $training_page ) . '#' . $post_cat->slug; ?>" class=""><?php
+			      echo $post_cat->name;
+
+			      if ( get_field( 'module_title', 'resource-category_' . $post_cat->term_id ) != '' ) {
+				      echo ': ' . get_field( 'module_title', 'resource-category_' . $post_cat->term_id );
+			      }
+			      ?></a>
+          </li>
+	      <?php
+      } ?>
+        </ol>
+    </nav>
+    
+    <?php
         
-        $post_cat = get_the_terms ( get_the_ID(), 'resource-category' );
         
-        $post_cat = $post_cat[0];
-          
-      ?>
+    ?>
     
-      <li class="breadcrumb-item"><a href="<?php echo get_permalink ( $training_page ) . '#' . $post_cat->slug; ?>" class=""><?php 
-        
-        echo $post_cat->name; 
+    <h1><?php
         
-        if ( get_field ( 'module_title', 'resource-category_' . $post_cat->term_id ) != '' ) {
-          echo ': ' . get_field ( 'module_title', 'resource-category_' . $post_cat->term_id );
+        if ( $hero_fields['title'] != '' ) {
+            echo $hero_fields['title'];
+        } else {
+            the_title();
         }
         
-      ?></a></li>
-      
-    </ol>
-  </nav>
-  
-  <?php
+    ?></h1>
     
+    <?php
+        
+        if ( 
+            get_field ( 'asset_type' ) == 'video' &&
+            get_field ( 'asset_video' ) != ''
+        ) {
+            
+    ?>
+    
+    <div id="asset-video">
+        <?php echo wp_oembed_get ( 'http://vimeo.com/' . get_field ( 'asset_video' ) ); ?>
+    </div>
     
-  ?>
-  
-  <h1><?php
+    <?php
+            
+        } else {
+            
+            echo apply_filters ( 'the_content', custom_excerpt ( get_the_ID(), 999 ) );
+            
+    ?>
+    
+    <div id="hero-btns" class="d-flex align-items-center">
+        <?php
+        
+            if ( get_field ( 'asset_time' ) != '' ) {
+                
+        ?>
+        
+        <div id="hero-asset-time" class="asset-time d-flex align-items-center mr-5">
+            <h6 class="mb-0 all-caps"><?php _e ( 'Time to completion', 'cdc' ); ?></h6>
+            <span class="border border-secondary text-white rounded-pill ml-4 py-2 px-4"><?php echo get_field ( 'asset_time' ); ?> min</span>
+        </div>
+        
+        <?php
+            
+            }
+            
+        ?>
     
-    if ( $hero_fields['title'] != '' ) {
-      echo $hero_fields['title'];
-    } else {
-      the_title();
-    }
+        <div id="hero-tts" class="d-flex align-items-center">
```

Comment:
The text to speech feature was indeed in main branch for a couple of days, but got reverted.

Reply:
Fixed!